src/main/java/org/example/utility/UtilityMethods.java

```diff
@@ -97,4 +97,13 @@ public static int countOdd(int[] array) {
         }
         return count;
     }
+
+    public static int findMin(int[] array){
+        checkNotNull(array, "Array is null.");
+        int min = array[0];
+        for (int i : array)
+            if (i < min)
```

Comment:
We should use curly braces for `if`, `for` and other statements. Please, have a look at the [Google Java Style Guide: 4.1.1 Braces are used where optional](https://google.github.io/styleguide/javaguide.html#s4.1.1-braces-always-used)

FYI: You can read about the motivation behind this rule: https://stackoverflow.com/questions/2125066/is-it-a-bad-practice-to-use-an-if-statement-without-curly-braces